### ISSUE TITLE
view gets rendered out of view bounds

### ISSUE THREAD

**Issue by noumantahir** [GuoChen-WHU/react-native-pinch-zoom-view#5]
while zooming in within a view box, the zoomed in view gets rendered even out side the view box and overlays other views
![screen shot 2017-08-26 at 8 52 19 pm](https://user-images.githubusercontent.com/6298342/29742874-80357f48-8aa0-11e7-9a74-50379f776880.png)

**Comment by sabonis** [comment]
You can set zIndex to its siblings.

**Comment by anniewey** [comment]
@noumantahir Did you find any solution?

**Comment by noumantahir** [comment]
no, I dropped it for webView rendering svg inside using a local .html file

**Comment by Antonides** [comment]
overflow hidden on the parent

**Comment by marsonmao** [comment]
overflow hidden is not working...

**Comment by marsonmao** [comment]
A very strange bug (?), you have to set both `overflow: 'hidden'` and `backgroundColor` to make it work. I tried `backgroundColor: 'transparent'` and it's also working.

**Comment by HAFDIAHMED** [comment]
up
